### PR TITLE
Added Default landing page@V0.0.2

### LINKED ISSUE #8: Feature : Add Default Landing Page
**After closing this issue site must have a landing _Home_ _Page_**
> **_Items checklist_**

- [x] **Heading**
- [x] **Title**
- [x] **Welcome Message**
- [x] **Sign In** 
- [x] **Register**
- [x] **Nav Ba**

### PR DESCRIPTION
**Pull request by myretro**
Closed #8

### CODE REVIEW

File: views/index.css [hunk 1]
```diff
@@ -1,19 +1,64 @@
+@import url("https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap");
+
 * {
   box-sizing: border-box;
-  margin: 0;
-  padding: 0;
+  padding: 0px;
```
Comment (abhinavk454) [inline views/index.css]: 0 has no unit

**Review by abhinavk454 — CHANGES_REQUESTED** [review]
Need to remove px from 0 values.

File: views/index.css (hunk 1 above)
Comment (myretro) [reply views/index.css:5]: review gain

**Review by abhinavk454 — APPROVED** [review]
Approved

**Pull request merged**